current_fall2025/scripts/test_vector_semetic_search.py

```
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
```

Ruff (0.14.0)

Shebang is present but file is not executable

(EXE001)